### PR TITLE
feat: update package.json exports

### LINKED ISSUE #41: Fix filename typo
fix `src/logger/transport-generator.spaec.ts` to `src/logger/transport-generator.spec.ts`

### PR DESCRIPTION
**Pull request by Ko1103**
resolves #41, #42

### CODE REVIEW

File: src/logger/transport-generator.spec.ts [hunk 1]
```diff
@@ -8,9 +8,6 @@ describe('TransportGenerator', () => {
       const transport = TransportGenerator.cli(level);
       expect(transport).toBeInstanceOf(transports.Console);
       expect(transport.level).toBe(level);
-      expect(transport.format).toEqual(
-        format.combine(format.timestamp(), format.colorize(), format.simple()),
-      );
```
Comment (Ko1103) [inline src/logger/transport-generator.spec.ts:13]: This doesn't work, but it's not necessarily essential, so I excluded it.

File: src/logger/transport-generator.spec.ts [hunk 2]
```diff
@@ -20,18 +17,12 @@ describe('TransportGenerator', () => {
       const transport = TransportGenerator.dd(level);
       expect(transport).toBeInstanceOf(transports.Console);
       expect(transport.level).toBe(level);
-      expect(transport.format).toEqual(
-        format.combine(format.timestamp(), format.json()),
-      );
```
Comment (Ko1103) [inline src/logger/transport-generator.spec.ts:25]: Here is the same

File: src/logger/transport-generator.spec.ts [hunk 3]
```diff
@@ -20,18 +17,12 @@ describe('TransportGenerator', () => {
...
     });
     it('returns a ConsoleTransportInstance with the given info for datadog', () => {
       const level = 'info';
       const transport = TransportGenerator.dd(level);
       expect(transport).toBeInstanceOf(transports.Console);
       expect(transport.level).toBe(level);
-      expect(transport.format).toEqual(
-        format.combine(format.timestamp(), format.json()),
-      );
     });
```
Comment (Ko1103) [inline src/logger/transport-generator.spec.ts:26]: Here is the same

**Review by ThaddeusJiang — APPROVED** [review]
LGTM

**Pull request merged**